Source/Core/VideoBackends/Metal/MTLMain.mm

```diff
@@ -156,11 +159,11 @@ static bool WindowSystemTypeSupportsMetal(WindowSystemType type)
 
 void Metal::VideoBackend::PrepareWindow(WindowSystemInfo& wsi)
 {
-  if (wsi.type != WindowSystemType::MacOS)
-    return;
```

Comment:
Can you leave this in?  I don't really expect to receive any other window types, but just in case